packages/gatsby-theme-docs/src/hooks/use-all-release-notes.js

```diff
@@ -0,0 +1,40 @@
+import { useStaticQuery, graphql } from 'gatsby';
+
+const useAllReleaseNotes = () => {
+  const queryResult = useStaticQuery(
+    graphql`
+      query GetAllReleaseNotesForFilters {
+        allReleaseNotePage(sort: { order: DESC, fields: date }) {
+          nodes {
+            date(formatString: "YYYY-MM-DD")
+            topics
+          }
+        }
+      }
+    `
+  );
+
+  return queryResult.allReleaseNotePage.nodes;
+};
+
+export default useAllReleaseNotes;
+
+export const useEarliestLatestDates = () => {
+  const allReleaseNotes = useAllReleaseNotes();
+  return {
+    latestDate: allReleaseNotes[0].date,
+    earliestDate: allReleaseNotes[allReleaseNotes.length - 1].date,
+  };
+};
+
+export const useReleaseNotesTopicsSet = () => {
```

Comment:
```suggestion
export const useReleaseNotesTopics = () => {
```

Then return `return Array.from(topics)`

Reply:
Can you rename this function without the `Set` suffix part?